filters/exclusions.txt

```diff
@@ -236,6 +236,7 @@ _campaign=$popup
 !###########################
 !
 !### xinggsf ###
+/(?<!m\.)(www\.)?youtube\.com(,)?.*#%#\/\/scriptlet\(/
```

Comment:
As far as I understand, it will disable all scriptlet rules for youtube from third party filter lists.
Currently I see scriptlets for youtube only in `xinggsf` filter, so it's not a problem, but in the future there are might be useful scriptlets in other lists as well.
So, maybe it would be better to exclude only these which are problematic?

Reply:
```suggestion
(?<!m\.)(www\.)?youtube\.com(,)?.*#%#\/\/scriptlet\(['"]{1}(ubo-)?json-prune[\.a-z'"]+, *['"]{1}(\[\]\.playerResponse\.adPlacements|\[\]\.playerResponse\.playerAds|playerResponse\.adPlacements|playerResponse\.playerAds|adPlacements\.playerAds)
```
https://regex101.com/r/3bBQWc/1

Like this? Or, just a literally the rule?

Reply:
original rule or a part can be added as is without `/regex/`

Reply:
```suggestion
#%#//scriptlet('ubo-json-prune.js', '[].playerResponse.adPlacements [].playerResponse.playerAds playerResponse.adPlacements playerResponse.playerAds adPlacements playerAds')
```
@Alex-302 Like this?

Reply:
If only a specific rule causes it, then I think that we can exclude one specific rule.
If there will be more rules causing detection, then we can use regexp.
But it's just my opinion, so maybe better to wait and see what other @AdguardTeam/filters-maintainers say.

Reply:
It is owned by @xinggsf.

Reply:
> Btw, can the author do this on his own?

A PR has been submitted but not yet merged: https://github.com/xinggsf/Adblock-Plus-Rule/pull/86
Thanks